### DISCUSSION TITLE
2.0是不是没有屏幕视频投票的功能?

### DISCUSSION THREAD

**Discussion by jiang-yuan** [the1812/Bilibili-Evolved#2697] [提问 / Q&A] [answered]
搜索了一圈设置找不到了

**Comment by Geeyun-JY3** [comment, marked as answer]
想说的是屏蔽视频的投票弹窗是吧

叫 `删除视频弹窗`：https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md#%E5%88%A0%E9%99%A4%E8%A7%86%E9%A2%91%E5%BC%B9%E7%AA%97
